src/pages/permissions/PermissionIdpGroups.tsx

```diff
@@ -0,0 +1,272 @@
+  );
+
+  const selectedGroups = groups.filter((group) =>
+    selectedGroupNames.includes(group.name),
+  );
+
+  const groupsForRows =
+    panelParams.panel && panelParams.panel !== panels.createIdpGroup
+      ? selectedGroups
+      : filteredGroups;
+
+  const rows = groupsForRows.map((idpGroup) => {
+    return {
+      name: idpGroup.name,
+      className: "u-row",
+      columns: [
+        {
+          content: idpGroup.name,
+          role: "cell",
+          "aria-label": "Name",
+          className: "u-truncate",
+          title: idpGroup.name,
+        },
+        {
+          content: idpGroup.groups.length,
+          role: "cell",
+          className: "u-align--right",
+          "aria-label": "Number of mapped groups",
+        },
+        {
+          className: "actions u-align--right",
+          content: <IdpGroupActions idpGroup={idpGroup} />,
+          role: "cell",
+          "aria-label": "Actions",
+        },
+      ],
+      sortData: {
+        name: idpGroup.name.toLowerCase(),
+        groups: idpGroup.groups.length,
+      },
+    };
+  });
+
+  const { rows: sortedRows, updateSort } = useSortTableData({ rows });
+
+  if (isLoading) {
+    return <Loader text="Loading identity provider groups" />;
+  }
+
+  const getTablePaginationDescription = () => {
+    if (panelParams.panel) {
+      return `Showing selected ${pluralize("IDP group", sortedRows.length)}`;
+    }
+
+    if (selectedGroupNames.length > 0) {
+      return (
+        <SelectedTableNotification
+          totalCount={groups.length ?? 0}
+          itemName="IDP group"
+          parentName=""
+          selectedNames={selectedGroupNames}
+          setSelectedNames={setSelectedGroupNames}
+          filteredNames={filteredGroups.map((item) => item.name)}
+        />
+      );
+    }
+  };
+
+  const hasGroups = groups.length > 0;
+  const content = hasGroups ? (
+    <ScrollableTable
+      dependencies={[groups]}
+      tableId="idp-groups-table"
+      belowIds={["status-bar"]}
+    >
+      <TablePagination
+        data={sortedRows}
+        id="pagination"
+        itemName="IDP group"
+        className="u-no-margin--top"
+        aria-label="Table pagination control"
+        description={getTablePaginationDescription()}
+      >
+        <SelectableMainTable
+          id="idp-groups-table"
+          headers={headers}
+          rows={sortedRows}
+          sortable
+          emptyStateMsg="No identity provider groups found matching this search"
+          onUpdateSort={updateSort}
+          itemName="IDP group"
+          parentName=""
+          selectedNames={selectedGroupNames}
+          setSelectedNames={setSelectedGroupNames}
+          processingNames={[]}
+          filteredNames={filteredGroups.map((item) => item.name)}
+          disableSelect={!!panelParams.panel}
+        />
+      </TablePagination>
+    </ScrollableTable>
+  ) : (
+    <EmptyState
```

Comment:
The empty state has the same explanatory copy twice

![image](https://github.com/canonical/lxd-ui/assets/1155472/2c3750cd-abb0-4c26-abbc-7d562d6c8485)

Reply:
Yes will remove that.